src/webview/leetCodePreviewProvider.ts

```diff
@@ -11,19 +11,40 @@ class LeetCodePreviewProvider extends LeetCodeWebview {
 
     private node: IProblem;
     private description: IDescription;
+    private sideMode: boolean = false;
 
-    public async show(node: IProblem): Promise<void> {
+    public isSideMode(): boolean {
+        return this.sideMode;
+    }
+
+    public async show(node: IProblem, isSideMode: boolean = false): Promise<void> {
         this.description = this.parseDescription(await leetCodeExecutor.getDescription(node), node);
         this.node = node;
+        this.sideMode = isSideMode;
         this.showWebviewInternal();
+        if (this.sideMode) {
+            // Hide the side bar for better view area
+            commands.executeCommand("workbench.action.focusSideBar").then(() => {
```

Comment:
This is not an elegant workaround...
But I searched for hours the api to check the side bar status but in vain...

Workarounds which would have been possible:
1. api in `vscode.window` namespace. We could control the visible state of things like terminal/text editor, but side bar isn't in the list.
2. context value. One [issue](https://github.com/Microsoft/vscode/issues/66956) in vscode suggests that `sidebarVisible` can be used for `toggleSideBar` to work like `closeSidebar`. Yet neither can I find out how to invoke `executeCommand()` with `when` context, nor can I retrieve the context value (there is a command named `setContext`, yet `getContext` does not exist)

Reply:
Wrap the following code into a function like `hideLeftSideBar()`, then no need to add comments
```
commands.executeCommand("workbench.action.focusSideBar").then(() => {commands.executeCommand("workbench.action.toggleSidebarVisibility");
```

Reply:
Updated